test/e2e/scenarios/Cluster/cluster_create_update.yaml

```diff
@@ -61,17 +61,16 @@ steps:
         conditions:
           ACK.ResourceSynced:
             status: "True"
-            timeout: 7200
-
-  - id: "USER_ONE_CREATE"
-    description: "Create User ONE"
+            timeout: 3600
```

Comment:
60 minutes? Is this an appropriate value?

Reply:
It has a rare chance that previous update of snapshot window causes a snapshotting process which could take a long time. All updates have to wait till cluster finishes snapshotting.